### ISSUE TITLE
FB8956392: Cannot set VoiceOver accessibilityLabel or accessibilityHint for Button in ToolbarItem

### ISSUE THREAD

**Issue by AngeloStavrow** [feedback-assistant/reports#188]
- **Date:** 2020-12-29
- **Resolution:** Open
- **Area:** SwiftUI Framework
- **OS:** iOS 14.4
- **Type:** Incorrect/Unexpected Behavior
- **Keywords:** swiftui, toolbar, accessibility, a11y

### Please describe the issue:
Attempting to add or change an accessibility label or hint for VoiceOver in a SwiftUI view doesn’t work when the Button is wrapped in a ToolbarItem.

(Tested in iOS 14.4)

### Please list the steps you took to reproduce the issue:

1. Open the attached sample project
2. Run the app on the device with VoiceOver on
3. Tap the buttons in the navigation and button bar

### What did you expect to happen?

I expect VoiceOver to speak the accessibility label and hint provided for each button.

### What actually happened?

VoiceOver ignores accessibility labels and hints that are provided.

## Files

[SwiftUIButtonA11ySample.zip](https://github.com/feedback-assistant/reports/files/5755539/SwiftUIButtonA11ySample.zip)
